### PR TITLE
ci(cypress): fix iatapay upi payments

### PR DESCRIPTION
**Pull request by likhinbopanna**
## Type of Change

- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement
- [ ] Refactoring
- [ ] Dependency updates
- [ ] Documentation
- [x] CI/CD

## Description

Fixed the IataPay Cypress test cases as UPI_Collect payments were failing due to an invalid billing.address.country.

### Additional Changes

- [ ] This PR modifies the API contract
- [ ] This PR modifies the database schema
- [ ] This PR modifies application configuration/environment variables

## Motivation and Context
Cypress test cases were failing 

## How did you test it?
Cypress 

Before
image

After
image

Adyen 
image

Bluesnap
image

## Checklist

- [ ] I formatted the code `cargo +nightly fmt --all`
- [ ] I addressed lints thrown by `cargo clippy`
- [x] I reviewed the submitted code
- [ ] I added unit tests for my changes where possible

### CODE REVIEW

File: cypress-tests/cypress/e2e/spec/Payment/00022-Variations.cy.js [hunk 1]
```diff
@@ -763,6 +763,21 @@ describe("Corner cases", () => {
...
+      cy.createConfirmPaymentTest(
+        fixtures.createConfirmPaymentBody,
+        data,
+        "no_three_ds",
+        "automatic",
+        globalState
+      );
+      if (shouldContinue) shouldContinue = utils.should_continue_further(data);
+    });
+
```
Comment (pixincreate) [inline cypress-tests/cypress/e2e/spec/Payment/00022-Variations.cy.js:780]: why is this required in variations all of a sudden?

since you've touched the spec file, i would suggest you running the test against at least one more connector that is not in the github ci. adyen may be?

Comment (likhinbopanna) [reply cypress-tests/cypress/e2e/spec/Payment/00022-Variations.cy.js:780]: Its needed for subsequent test cases as iatapay doesn't have card payment method , running other test cases

**Review by pixincreate — APPROVED** [review]

**Review by Gnanasundari24 — APPROVED** [review]

**Comment by pixincreate** [comment]
ci check for cypress failed again, can you check @likhinbopanna ?
